cisa-advisory/info.json

```diff
@@ -0,0 +1,407 @@
+    "publisher": "",
+    "icon_small_name": "",
+    "icon_large_name": "",
+    "configuration": {
+        "fields": []
+    },
+    "operations": [
+        {
+            "operation": "get_ics_advisory",
+            "title": "Get ICS Advisory",
+            "annotation": "get_ics_advisory",
+            "description": "Fetches all the ICS Advisory till present date or by provided date filter.",
+            "category": "investigation",
+            "is_config_required": true,
+            "visible": true,
+            "enabled": true,
+            "parameters": [
+                {
+                    "name": "date_filter",
+                    "title": "Date Filter",
+                    "type": "select",
+                    "editable": true,
+                    "visible": true,
+                    "required": false,
+                    "description": "Specify the date filter.",
+                    "options": [
+                        "Today",
+                        "Last 7 Days",
+                        "Last 30 Days",
+                        "This Month",
+                        "This Year",
+                        "Last Month",
+                        "Last 3 Months",
+                        "Last 6 Months",
+                        "Last Date Published Advisory"
+                    ]
+                }
+            ],
+            "output_schema": {
+                "url": "",
+                "name": "",
+                "icsa_code": "",
+                "year": 0,
+                "vendor": "",
+                "product": "",
+                "cve_list": [],
+                "cwe_list": [],
+                "release_date": "",
+                "updated_date": "",
+                "cvss_severity": "",
+                "cvss_score": "",
+                "affected_product": "",
+                "company_headquaters_location": "",
+                "country_area_deployed": "",
+                "critical_infrastructure_sector": ""
+            }
+        },
+        {
+            "operation": "get_ics_advisory_by_vendor",
+            "title": "Get ICS Advisory By Vendor",
+            "annotation": "get_ics_advisory_by_vendor",
+            "description": "Fetches ICS Advisory on the basis of specified vendor name.",
+            "category": "investigation",
+            "is_config_required": true,
+            "visible": true,
+            "enabled": true,
+            "parameters": [
+                {
+                    "name": "vendor",
+                    "title": "Vendor",
+                    "type": "text",
+                    "editable": true,
+                    "visible": true,
+                    "required": true,
+                    "description": "Specify the vendor to get ICS Advisory "
+                }
+            ],
+            "output_schema": {
+                "url": "",
+                "name": "",
+                "icsa_code": "",
+                "year": 0,
```

Comment:
please make year as empty in all output schema

Reply:
Done